src/bp/ui-studio/src/web/actions/index.js

```diff
@@ -165,7 +167,9 @@ export const fetchBotInformation = () => dispatch => {
 
 export const botsReceived = createAction('BOTS/RECEIVED')
 export const fetchAllBots = () => dispatch => {
-  axios.get(`${window.API_PATH}/admin/bots`).then(res => dispatch(botsReceived(res.data)))
+  axios
+    .get(`${window.API_PATH}/admin/bots`, { headers: { 'X-BP-Workspace': 'default' } })
+    .then(res => dispatch(botsReceived(res.data)))
```

Comment:
I think that route should stay restricted to the admin UI. The workspace can't be hardcoded, since when there will be another one, it will fail. The workspace is discovered from the botId in an express middleware.

I suggest adding a new route available on the studio which would return only the bot IDs (not the whole config file for each bots)

Reply:
Are bot ids unique for each workspace or per instance?
I wasn't sure to understand since we require the header but couldn't figure out where to retrieve it from the botId.
We could probably just pass that info in the studio store and use it in the same actions?

Reply:
Bot ID are unique for the whole server, and they can only be assigned to a workspace. So, there is a method on workspaceService to get the Workspace id from the bot id.

I'm not sure the studio should be aware of its workspace, since it doesn't need it for its operation